src/magneticod/persistence.go

```diff
@@ -226,3 +227,30 @@ func setupSqliteDatabase(database *sql.DB) error {
 
 	return nil
 }
+
+func setupMySQLDatabase(database *sql.DB) error {
+	_, err := database.Exec(
+		`CREATE TABLE IF NOT EXISTS torrents ("
+			id		INTEGER PRIMARY KEY AUTO_INCREMENT,
+			info_hash	BINARY(20) NOT NULL UNIQUE,
+			name		VARCHAR(255) NOT NULL,
```

Comment:
Not very familiar with MySQL but this would limit the length of name to 255 *bytes*, right? (not even Unicode Code Points)